redux/saga.js

```diff
@@ -3,7 +3,7 @@ import { all, call, delay, put, takeLatest } from 'redux-saga/effects'
 import { actionTypes, fetchArchives, fetchData } from './store'
 
 // Use this for mocking only.
-// import { fatalities } from '../mock-api/mock-data'
+import { fatalities } from '../mock-api/mock-data'
```

Comment:
Comment this out.

Reply:
And as you said, don't forget to re-enable the GET request in the saga.

Reply:
Commenting complete.

Looking into re-enabling GET request.

Reply:
Completed.